draft-ietf-quic-tls.md

```diff
@@ -618,11 +618,16 @@ application data, a server MUST treat receiving early data on the TLS connection
 as a connection error of type PROTOCOL_VIOLATION.
 
 
-## Rejecting 0-RTT
+## Accepting and rejecting 0-RTT
 
-A server rejects 0-RTT by rejecting 0-RTT at the TLS layer.  This also prevents
-QUIC from sending 0-RTT data. A server will always reject 0-RTT if it sends a
-TLS HelloRetryRequest.
+A server accepts 0-RTT by sending the EarlyDataIndication in the ServerHello
+(see Section 4.10.2 of {{!TLS13}}). A server that accepts 0-RTT MUST process and
+acknowledge all 0-RTT packets that it receives.
```

Comment:
```suggestion
acknowledge all valid 0-RTT packets that it receives.
```

Reply:
How would you even acknowledge an invalid packet?

Reply:
Fair point, but that only makes it harder to comply.